locales/en.json

```diff
@@ -1,4 +1,7 @@
 {
+  "45": "45",
+  "50": "50",
+  "4566": "4566",
```

Comment:
Can we delete these numbers?

Reply:
Oops. I'll have to clean this up in a new PR. Maybe I'll bundle it in with the test cleanup issue?